docs/intro/contributing.txt

```diff
@@ -117,38 +117,41 @@ Download the Django source code repository using the following command:
 
 Now that you have a local copy of Django, you can install it just like you would
 install any package using ``pip``. The most convenient way to do so is by using
-a *virtual environment* (or virtualenv) which is a feature built into Python
+a *virtual environment* which is a feature built into Python
 that allows you to keep a separate directory of installed packages for each of
 your projects so that they don't interfere with each other.
 
-It's a good idea to keep all your virtualenvs in one place, for example in
-``.virtualenvs/`` in your home directory. Create it if it doesn't exist yet:
+It's a good idea to keep all your virtual environments in one place, for example
+in ``.virtualenvs/`` in your home directory.
 
-.. console::
-
-    $ mkdir ~/.virtualenvs
-
-Now create a new virtualenv by running:
+Create a new virtual environment by running:
```

Comment:
How about "Create `~/.virtualenvs` and a new virtual environment within by running:" ?

Reply:
hmm I think it's redundant